### ISSUE TITLE
Pidgin install

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/windows-package-manager#195]
```
What steps will reproduce the problem?
1. Install pidgin
2. Run pidgin
3.

What is the expected output? What do you see instead?
 The program can't start because libatk-1.0-0.dl is missing from your computer. Try reinstalling the program to fix this problem.
and...
Error loading pidgin.dll. Error. (126) The specified module could not be found. 
This probably means that GTK+ can't be found

What version of the product are you using? On what operating system?
version 2.10.6 on windows 7 64 bit.


Please provide any additional information below.

Thanks

```

Original issue reported on code.google.com by `cmtopi...@gmail.com` on 29 Jan 2013 at 8:58
* Merged into: #194

**Comment by GoogleCodeExporter** [comment]
```
the DLL should be inside the program directory. I have it here:
C:\ProgramFiles\Pidgin\Gtk\bin\libatk-1.0-0.dll
Could you please check that you have the file?
```

Original comment by `tim.lebe...@gmail.com` on 30 Jan 2013 at 6:28

**Comment by GoogleCodeExporter** [comment]
```
I don't get a GTK directory after install.
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
I know what this is. Pidgin site is blocked by our proxy. Pidgin installer
gets dependency from it's site. It might be better not to rely on
dependencies outside of our own npackd repos.  Thanks
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 8:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.lebe...@gmail.com` on 31 Jan 2013 at 5:53
* Changed state: **Duplicate**
